Source/Painting/SvgFallbackPaintServer.cs

```diff
@@ -9,6 +9,7 @@ namespace Svg
     /// <summary>
     /// A wrapper for a paint server has a fallback if the primary server doesn't work.
     /// </summary>
+    [Obsolete("Will be removed.")]
```

Comment:
It would be nice to have an explanation or something like "Use xxx instead" as a message instead of "Will be removed" (which is already the meaning of `Obsolete`).